src/interactive/plots.ts

```diff
@@ -212,6 +212,10 @@ export function displayPlot(params: { kind: string, data: string }) {
                         actions: false
                     }
                     var spec = ${payload}
+                    if (!(spec.width || spec.height)) {
+                        spec.width = 'container'
+                        spec.height = 'container'
+                    }
```

Comment:
Hm, I think this is a bit too magical at this point here, I don't think we should modify the spec itself. Or minimally I think this would need to be behind an option that is off by default. I think I could consider making this some sort of default in VegaLite.jl itself, maybe...

Reply:
Hm, if the user doesn't specify a height or width then they clearly don't care about it much and we can offer a better experience than just using VegaLite's defaults, no?

Reply:
No, I think if they don't specify it they assume that it will be the default size, the same size that they would get if they save the plot. What I really would want to avoid is that one generates a different plot depending on whether it gets shown in the UI or saved to disc, from the same plot command.

Reply:
Fair enough. Removed the auto-resizing for Vega then.